src/Altinn.Apps/AppTemplates/AspNet/App.IntegrationTests/CommonTests/RequestHandling/RequestPartValidatorTest.cs

```diff
@@ -0,0 +1,252 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+using Altinn.App.Common.RequestHandling;
+using Altinn.Platform.Storage.Interface.Models;
+
+using App.IntegrationTests.Utils;
+
+using Xunit;
+
+namespace App.IntegrationTestsRef.CommonTests.RequestHandling
+{
+    public class RequestPartValidatorTest
+    {
+        [Fact]
+        public void ValidatePart_InvalidDataType()
+        {
+            // Arrange
+            Application app = TestDataUtil.GetApplication("ttd", "events");
+            string partName = "kattebilde";
+
+            RequestPart part = new RequestPart
+            {
+                Name = partName
+            };
+
+            string expectedErrorMessage = $"Multipart section named, '{partName}' does not correspond to an element type in application metadata";
```

Comment:
Two simple tips. 
- Move declaration and initialization as close to first use of the variable as possible.
- Constants can be constant